server/src/main/java/com/flipkart/varadhi/VaradhiApplication.java

```diff
@@ -0,0 +1,131 @@
+package com.flipkart.varadhi;
+
+import com.flipkart.varadhi.cluster.ClusterManager;
+import com.flipkart.varadhi.cluster.impl.ClusterManagerImpl;
+import com.flipkart.varadhi.components.Component;
+import com.flipkart.varadhi.components.ComponentKind;
+import com.flipkart.varadhi.components.controller.Controller;
+import com.flipkart.varadhi.components.server.Server;
+import com.flipkart.varadhi.config.AppConfiguration;
+import com.flipkart.varadhi.exceptions.InvalidConfigException;
+import com.flipkart.varadhi.utils.HostUtils;
+import io.vertx.config.ConfigRetriever;
+import io.vertx.config.ConfigRetrieverOptions;
+import io.vertx.config.ConfigStoreOptions;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.VertxOptions;
+import io.vertx.core.json.JsonObject;
+import io.vertx.micrometer.MicrometerMetricsOptions;
+import io.vertx.tracing.opentelemetry.OpenTelemetryOptions;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class VaradhiApplication {
```

Comment:
I personally liked Server better :) concise and in the server module so kinda obvious what it was.